src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ValidateVariants.java

```diff
@@ -27,8 +27,8 @@
 /**
  * Validate a VCF file with a strict set of criteria
  *
- * <p>This tool is designed to validate the correctness of the formatting of VCF files. In addition to standard
- * adherence to the VCF specification, this tool performs extra strict validations to ensure that the information
+ * <p>This tool is designed to validate the correctness of the formatting of VCF files. Besides standard
```

Comment:
@MartonKN, be sure to use the Edit button to be able to copy-paste the html formatting that Markdown is currently translating.

--> This tool is designed to validate the adherence of a file to VCF format. The tool will validate .g.vcf GVCF format files as well. For VCF specifications, see <a href='https://samtools.github.io/hts-specs/'>https://samtools.github.io/hts-specs/</a>.

Reply:
OK